hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -1427,7 +1442,11 @@ public boolean isConsistentLogicalTimestampEnabled() {
   }
 
   public Boolean shouldAutoCommit() {
-    return getBoolean(AUTO_COMMIT_ENABLE);
+    return false;
+  }
+
+  public Boolean shouldInternalAutoCommit() {
```

Comment:
Can we make this return a primitive boolean?

Reply:
we have removed this completely. we took a slightly diff approach.